### PR TITLE
Remove older receipt setting fields related to receipt templates

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/5368

⚠️ Depends on, https://github.com/opencollective/opencollective-pdf/pull/844

### CODE REVIEW

File: components/Receipt.js
Comment (Betree) [inline components/Receipt.js:341]: Shouldn't we deprecate https://github.com/opencollective/opencollective-api/blob/8571d308acb74cc70fa6108e09884717610c7a11/server/graphql/v1/queries.js#L182 if we're doing that?

Comment (SudharakaP) [reply components/Receipt.js:341]: I've marked these `title` and `extraInfo` fields are deprecated in the relavant API PR,

https://github.com/opencollective/opencollective-api/blob/f86e7e6e31dab8a4f28aa0aede526827625b92ff/server/graphql/v1/types.js#L633

https://github.com/opencollective/opencollective-api/blob/f86e7e6e31dab8a4f28aa0aede526827625b92ff/server/graphql/v1/types.js#L642

Later once the API PR and this one is merged we should I think remove these fields as well from the API. 🤔

**Review by Betree — APPROVED** [review]
Good for me, I'll let @kewitz provide a final review

**Pull request merged**
